ui/src/global.scss

```diff
@@ -187,6 +187,24 @@ ion-button {
   }
 }
 
+.action-success-modal {
+  .modal-wrapper {
+    width: 320px !important;
+    height: 320px !important;
+    top: unset !important;
+    left: unset !important;
+  }
+}
+
+.action-success-modal-qr {
```

Comment:
This feels a bit messy, especially if additional classes are needed for @ media queries for screen size. Have you tried just un-setting our other gloabl .modal-wrapper adjustments. In other words, instead of styling on top of styling, you may be able to get the modal you want by removing the first styling. Maybe not.